corehq/apps/sms/models.py

```diff
@@ -2396,6 +2397,13 @@ def get_sms_rate_limit(self):
     def send(self, msg, *args, **kwargs):
         raise NotImplementedError("Please implement this method.")
 
+    @classproperty
+    def using_api_to_get_fees(cls):
+        """
+        Override in case backend is fetching gateway fees through provider API
+        """
+        return False
```

Comment:
Why `classproperty` instead of a class attribute?

```suggestion
    # Override in case backend is fetching gateway fees through provider API
    using_api_to_get_fees = False
```